### PR TITLE
Update CSS: A blog post

### PR DESCRIPTION
**Pull request by thebeatlesphan**
image

### CODE REVIEW

File: app/[blog]/page.js [hunk 1]
```diff
@@ -1,6 +1,59 @@
-const Blog = () => {
-  // TODO
-  return <div>Blog Page</div>;
+import Image from "next/image";
+
+import kid from "../../public/kid.png";
+import styles from "./page.module.css";
+
+const Blog = (props) => {
```
Comment (HamzahMansour) [inline app/[blog]/page.js:6]: should this be moved to `BlogPost`? here: https://github.com/Hack-Weekly/tangerine-moose-blog/blob/7e4142a160fa866da54f33faca6eb15b85954ea9/app/%5Bblog%5D/%5Bpost%5D/page.js#L122

Comment (thebeatlesphan) [reply app/[blog]/page.js:6]: Yeah I just noticed that. So is that the Blog Post? 
What page was I working on?

Comment (HamzahMansour) [reply app/[blog]/page.js:6]: Blog here would be the list of all blog posts as cards, user info/bio at the top. so i think the url would be `/[blogName]` and blog posts are linked here as `/[blogName]/[blogPostSlug]`

Comment (HamzahMansour) [reply app/[blog]/page.js:6]: so i guess the about the author section would stay, and the blog post would go into a card with the text cut off

Comment (thebeatlesphan) [reply app/[blog]/page.js:6]: Ok.

I'll just make the blog home page then.
Your blog post has a lot going right now haha

**Comment by thebeatlesphan** [comment]
Working on moving to /[blog]/[slug]

**Pull request closed without merging**